src/view.js

```diff
@@ -191,7 +198,10 @@ export class View {
           this.#currentSelectedIntervalSemitones = semitones;
           this.#toggleSecondaryButtonState('semitones', semitones);
           const enableSubmitButtonTimer = setInterval(() => {
-            if (!playTonesButton.disabled && !getHintButton.disabled) {
+            if (
+              !playTonesButton.disabled &&
+              (this.#level === 'Hard' || !getHintButton.disabled)
+            ) {
```

Comment:
Just tested it out after the changes were merged from main - I think we over-complicated this a bit previously, and this breaks the intermediate level.

Since we disable both buttons anyway when either are clicked, it's enough just to check whether the "Play tones" button is disabled.

Other than this I think we're good to merge!
```suggestion
            if (!playTonesButton.disabled) {
```

Reply:
I read it through and you are right. I will update the file and gp.
By the way, I was just skimming through the `view.js` file, especially the CSS-related parts you wrote; your codes are so efficient and organized! I hope I could write like you soon ❤️